src/rospack.cpp

```diff
@@ -80,7 +80,8 @@
 #define PyUnicode_FromString PyString_FromString
 #endif
 
-using namespace tinyxml2;
+// use namespace to disambiguate from MSXML on Windows
+namespace tx2 = tinyxml2;
```

Comment:
Instead of defining another namespace (`tx2`) please use the original namespace instead (`tinyxml2`).